pkg/bpf/bpffs.go

```diff
@@ -29,8 +30,13 @@ var (
 
 	// Set to true on first get request to detect misorder
 	lockedDown = false
+	once       sync.Once
```

Comment:
Are you talking of the race conditions between line 41 and the others like 48 for example?